apollo-http-cache/src/main/kotlin/com/apollographql/apollo3/cache/http/DiskLruHttpCache.kt

```diff
@@ -43,6 +43,10 @@ class DiskLruHttpCache(private val fileSystem: FileSystem, private val directory
         .build()
   }
 
+  /**
+   * Store the [response] with the given [cacheKey] into the cache.
+   * Note: the response's body is not consumed nor closed.
```

Comment:
Wait, how come the response body is not consumed and closed? Since `write` needs to write it to the cache?

Reply:
Because I used `.peek()` and removed the `close()` :)

Reply:
Ah yes, sorry I missed that 👍